studio/stores/authConfig/schema/AuthProviders/AuthTemplatesValidation.tsx

```diff
@@ -127,6 +127,7 @@ export const EMAIL_CHANGE: FormSchema = {
 - \`{{ .Email }}\` : The original user's email address
 - \`{{ .NewEmail }}\` : The user's new email address
 - \`{{ .Data }}\` : The user's \`user_metadata\`
+- \`{{ .SendingTo }}\: The recipient of the email. Relevant for Secure Email Change.`
```

Comment:
maybe a link to the docs explaining why it's relevant when the "Secure Email Change" setting is enabled would be more useful here